docs/contributing.md

```diff
@@ -121,3 +121,75 @@ Prerequisites: dpkg -i http://archive.ubuntu.com/ubuntu/pool/universe/c/checksec
 Always run make sdl-check command before you submit changes.
 
 Prerequisites: pip3 install bandit
+
+## Logging Policy
+
+### Levels
+
+</br>
+
+#### `ERROR`
+Log at this level errors of every ovms operation that failed 
```

Comment:
What do you mean by "failed"? 
Handling of the client's requests can fail (wrong shape, inputs etc.) and OVMS can potentially handle thousands of requests. If someone would flood OVMS with badly formed requests we would see nothing but errors in logs. 

Perhaps we could define that on ERROR log level we should log only errors that are related to the internal model server state (like when a model could not be loaded for some reason). 
And errors that are results of handling users requests are logged on DEBUG level?